plugins/json_importer.cc

```diff
@@ -68,14 +71,46 @@ JSONImporterConfigDescription()
     addField("limit", &JSONImporterConfig::limit,
              "Maximum number of lines to process");
     addField("offset", &JSONImporterConfig::offset,
-            "Skip the first n lines.", int64_t(0));
+             "Skip the first n lines.", int64_t(0));
     addField("ignoreBadLines", &JSONImporterConfig::ignoreBadLines,
              "If true, any line causing an error will be skipped. Any line "
              "with an invalid JSON object will cause an error.", false);
+    addField("where", &JSONImporterConfig::where,
```

Comment:
Nvm. I changed it to push the data via the rowScope instead, which is likely the way it was intended tp be.